bytes/bytes/raw/file_raw_repository.py

```diff
@@ -62,3 +72,68 @@ def _raw_file_path(self, raw_id: UUID, boefje_meta: BoefjeMeta) -> Path:
 
     def _index(self, raw_id: UUID) -> str:
         return str(raw_id)[: self.UUID_INDEX]
+
+
+class S3RawRepository(RawRepository):
+    def __init__(
+        self,
+        file_middleware: FileMiddleware,
+        bucket_per_org: bool,
+        s3_bucket_prefix: str | None = "OpenKAT-",
+        s3_bucket_name: str | None = "OpenKAT",
```

Comment:
You probably want a default (even empty) value, but not a `None`? This could lead to weird formats below if someone passes a `None` explicitly

Reply:
I meant to write this as "You do not have to give this argument, and if you do not, we will assign something to it". I will refactor it, thank you 👍

Reply:
Ah I see. But then in that case you could simply remove the `None` type and give it a default value; which makes this then an optional keyword argument with a default